src/index.js

```diff
@@ -14,7 +14,7 @@ export const RCComponent = ({
   setClosableState,
   moreOpts = false,
   width = '100%',
-  height = '30vh',
+  height = '50vh',
```

Comment:
why do we need this change?

Reply:
![Screenshot 2023-03-06 185229](https://user-images.githubusercontent.com/80944218/223283532-e95e79c8-e0eb-45f0-98a5-0170d420e7f0.png)

For some reason, the recent changes caused the component height of 30 vh to make the chat window look very small, making it hard to conduct debugging, as shown in the screenshot above. So I changed it to 50 vh. Let me know if it's better to keep it at 30vh.

Reply:
No lets merge this, we need to think of a better way for making it compact.